packages/sdk/src/types/internal/context.ts

```diff
@@ -317,6 +320,26 @@ export class InternalContext {
         }
     }
 
+    public setSoundState(
+        soundInstance: SoundInstance,
+        options?: SetSoundStateOptions,
+        command?: SoundCommand,
+        soundAssetId?: string,
+        startTimeOffset?: number
+    ) {
+
+        this.context.internal.protocol.sendPayload(

```

Comment:
>this.context.internal.protocol.sendPayload [](start = 8, length = 42)

You're already in the internal context, so: `this.protocol.sandPayload` is sufficient.